studio/src/pages/account/manage.tsx

```diff
@@ -0,0 +1,162 @@
+import { UserContext } from "@/components/app-provider";
+import { getDashboardLayout } from "@/components/layout/dashboard-layout";
+import { Button } from "@/components/ui/button";
+import {
+  Card,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import {
+  Dialog,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/components/ui/dialog";
+import { Input } from "@/components/ui/input";
+import { useToast } from "@/components/ui/use-toast";
+import { useZodForm } from "@/hooks/use-form";
+import { NextPageWithLayout } from "@/lib/page";
+import { useMutation } from "@connectrpc/connect-query";
+import { EnumStatusCode } from "@wundergraph/cosmo-connect/dist/common/common_pb";
+import { deleteUser } from "@wundergraph/cosmo-connect/dist/platform/v1/platform-PlatformService_connectquery";
+import { useRouter } from "next/router";
+import { useContext, useState } from "react";
+import { z } from "zod";
+
+const ManageAccountPage: NextPageWithLayout = () => {
+  const user = useContext(UserContext);
+  const router = useRouter();
+  const [open, setOpen] = useState(false);
+
+  const schema = z.object({
+    email: z.string().regex(new RegExp(`^${user?.email}$`), {
+      message: "Please enter your email as requested.",
+    }),
+    confirmation: z.string().regex(new RegExp(`^DELETE MY ACCOUNT$`), {
+      message: "Please enter the confirmation correctly",
+    }),
+  });
+
+  type DeleteAccountInput = z.infer<typeof schema>;
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isValid },
+  } = useZodForm<DeleteAccountInput>({
+    schema,
+    mode: "onChange",
+  });
+
+  const { mutate, isPending } = useMutation(deleteUser);
+
+  const { toast } = useToast();
+
+  const handleDeleteUser = () => {
+    mutate(
+      {},
+      {
+        onSuccess: (d) => {
+          if (d.response?.code === EnumStatusCode.OK) {
+            toast({
+              description: "Deleted successfully.",
+              duration: 3000,
+            });
+            router.replace("/");
+          } else if (d.response?.details) {
+            toast({ description: d.response.details });
+          }
+        },
+        onError: () => {
+          toast({
+            description: "Could not delete the account. Please try again.",
+            duration: 3000,
+          });
+        },
+      },
+    );
+    setOpen(false);
+  };
+
+  return (
+    <Card className="border-destructive">
+      <CardHeader className="gap-y-6 md:flex-row">
+        <div className="space-y-1.5">
+          <CardTitle>Delete Account</CardTitle>
+          <CardDescription className="text-sm text-muted-foreground">
+            Your account will be permanently deleted. This action is
+            irreversible! <br /> You will lose memberships to all your
```

Comment:
You wanted to a section to explain what informations get lost in case of submission.

Reply:
I have explained further. I think it should be sufficient

Reply:
Fine for me, I thought we would explain the details as well before they run into an issue e.g. you can't delete yourself from orgs with more than one member when you are the only admin.